rfcs/shared/build-system/triage-automation.md

```diff
@@ -0,0 +1,223 @@
+>
+> - this is already implement via [github action](https://github.com/microsoft/fluentui/pull/24911)
+
+### 2. package label assignment
+
+> Note:
+>
+> - by "package" we mean official package name used to publish to npm registry without `@scope` prefix
+> - by "Component name" we mean official package name transformed via following `toPascalCase(packageName.replace('@fluentui/react-'))`
+
+**Workflow:**
+
+Based on selected project within issue template, additional select should be dynamically generated so contributor can pick proper package.
+
+<img width="569" alt="example of flow" src="https://user-images.githubusercontent.com/1223799/190431945-56fcd343-6eb7-4145-b2b2-62f5919b8c31.png">
+
+Unfortunately **[dynamic input controls are not possible with github issues beta](https://github.com/community/community/discussions/29067)** which leaves us with following options how to implement this:
+
+1. use only package names or Component name in "Library" picker
+
+- cons
+  - huge list because we have quite a lot packages
+- pros
+  - only 1 issue/feature template that we need to maintain
+  - removes need of `1. project label assignment` as labeling would be handled in one step
+
+2. creating feature/bug templates per library
+
+- cons
+  - 2 (feature/bug) x N templates (N is number of libraries in monorepo)
+- pros
+  - clear separation that can be tweaked to particular library needs if needed
+
+3. creating completely custom issue form solution (example [vuejs](https://github.com/vuejs/vue/issues/new/choose))
+
+- cons
+  - big effort in terms of development and maintenance
+- pros
+  - complete control about issue reporting
+  - another possible dog food for v9 controls
+
+**Workflow approach - summary:**
+
+After feedback and additional discussion the majority is in favor of "2. creating feature/bug templates per library", where the select list will contain "Component names".
+
+Simplified example of select box option for project issue template:
+
+```
+Component:
+ - Text
+ - Menu
+ - Dialog
+ - ...
+```
+
+Based on this, all project labels should adhere to unified pattern based on issue origin(project):
+
+1. issue origin: v9 template:
+
+```
+Text -> package: react-text, Fluent UI react-components (v9)
+Menu -> package: react-menu, Fluent UI react-components (v9)
+Dialog -> package: react-dialog, Fluent UI react-components (v9)
+```
+
+2. issue origin: v8 template:
+
+```
+Text -> package: react, Fluent UI react (v8)
+Menu -> package: react, Fluent UI react (v8)
+Dialog -> package: react, Fluent UI react (v8)
+Chart -> package: react-charting, Fluent UI react (v8)
+```
+
+3. issue origin: v0 template:
+
+```
+Text -> package: react-northstar, Fluent UI react-northstar (v0)
+Menu -> package: react-northstar, Fluent UI react-northstar (v0)
+Dialog -> package: react-northstar, Fluent UI react-northstar (v0)
+```
+
+4. issue origin: web-components template:
+
+```
+Text -> package: web-components
+Menu -> package: web-components
+```
```

Comment:
For cases 3. and 4. above, will we still have a label indicating the component? These have been pretty helpful for v8.

Reply:
- web-components doesn't use this https://github.com/microsoft/fluentui/issues?q=is%3Aissue+label%3Aweb-components
- v8, I think we can create pluggable architecture for the bot so these specific use cases can be handled solely for `@fluentui/react`. Do those labels mirror sub-folders or what is the logic ?